syncr_backend/network/handle_frontend.py

```diff
@@ -474,18 +475,36 @@ def handle_input_name(
     :param conn: socket.accept() connection
     :return: None
     """
-    if request['drop_name'] is None:
+    # This code assumes that the user has already
+    # created a folder to initialize as a drop.
+    # Work needs to be done on the frontend to
+    # instruct the user on initializing a drop.
+    #
+    # First, we check to see if Logger already contains name of drop id
+    # If so, do nothing, send message failure back
+    #
+    # Else, initialize directory with 'drop_name' as the name of the drop
+
+    # TODO: Allow user to select directory location from UI.
+
+    directory = request['directory']
+    drop_name = os.path.basename(directory)
+
+    # TODO: Change 'False' to check if drop_name already exists in tracker
+    if False:
         response = {
             'status': 'error',
-            'error': ERR_INVINPUT,
+            'result': 'failure',
+            'message': 'A drop already exists with the given drop name',
         }
     else:
-        # TODO: backend logic to create a drop.
-        # TODO: Test if given drop_name is valid.
+
+        initialize_drop(directory)
```

Comment:
Can this raise errors?  You probably want to handle them and return an error if an exception was raised.  Also does it return anything?